pages/dashboard/v2/[id].js

```diff
@@ -58,35 +58,52 @@ export async function getServerSideProps(context) {
     }
   });
 
-  let formattedStudentDataResponse = await formattedStudentData();
-
-  let timestamps = getCompletionTimestamps(formattedStudentDataResponse);
-
   let superblockURLS = await getDashedNamesURLs(
     certificationNumbers.fccCertifications
   );
 
-  let superBlockJsons = await getSuperBlockJsons(superblockURLS);
-  let dashboardObjs = createDashboardObject(superBlockJsons);
-  let totalChallenges = getTotalChallenges(dashboardObjs);
+  let superBlockJsons = await getSuperBlockJsons(superblockURLS); // this is an array of urls
```

Comment:
Instead of a comment letting us know this is an array of URLs, could we make this variable more descriptive so that it describes what it is supposed to be?

Reply:
Hi @GuillermoFloresV this name choice was introduced prior to me joining FCC which is why I added the comment. I could also make a separate 'easy' issue for this for CTI students or update it, @utsab let me know what you would prefer.

Reply:
https://github.com/freeCodeCamp/classroom/issues/495